core/src/main/resources/org/apache/spark/ui/static/spark-dag-viz.js

```diff
@@ -282,11 +282,7 @@ function renderDagVizForJob(svgContainer) {
 
 /* Render the dot file as an SVG in the given container. */
 function renderDot(dot, container, forJob) {
-  var escaped_dot = dot
-    .replace(/&lt;/g, "<")
-    .replace(/&gt;/g, ">")
-    .replace(/&quot;/g, "\"");
-  var g = graphlibDot.read(escaped_dot);
+  var g = graphlibDot.read(dot);
```

Comment:
So, did we handle all callers of `renderDot` in this PR?

Reply:
Ah, I've noticed StagePage needs those escaping.

Reply:
In the current implementation, a source text is used for tooltip text as HTML and used for node label  as plain text for DAG Viz.
I changed the usage of the source text for node label to interpret it as HTML.
So, I believe these escaping I remove from `renderDot` are not needed.